doc/nrf/getting_started/modifying.rst

```diff
@@ -164,12 +184,34 @@ Each child image is a separate application.
 
 For more information, see :ref:`ug_multi_image`.
 
-Changing the configuration temporarily
-======================================
+Changing the hardware configuration
+===================================
+
+To correctly edit the :file:`.dts` and :file:`.overlay` files for your project, you need to be familiar with the :ref:`Devicetree language and syntax <zephyr:dt-guide>`.
+
+.. tabs::
+
+   .. group-tab:: nRF Connect for VS Code
+
+      The |nRFVSC| offers several layers of `Devicetree integration <Devicetree support overview_>`_, ranging from summarizing devicetree settings in a menu and providing devicetree language support in the editor, to the Devicetree Visual Editor, a GUI tool for editing devictree files.
+      Follow the steps in `How to create devicetree files`_ and use one of the following options to edit the :file:`.dts` and :file:`.overlay` files:
+
+      * `Devicetree Visual Editor <How to work with Devicetree Visual Editor_>`_
+      * `Devicetree language support`_
+
+
+Changing the software configuration
+===================================
+
+You can change the software-related configuration temporarily or permanently.
+The temporary build files are deleted when you clean the build directory with the ``pristine`` target (see Zephyr's :ref:`zephyr:application_rebuild` for more information).
+
+Temporary changes
+-----------------
 
 When building your application, the different :file:`.config`, :file:`*_defconfig` files and the :file:`prj.conf` file are merged together and then processed by Kconfig.
 The resulting configuration is written to the :file:`zephyr/.config` file in your :file:`build` directory.
-This means that this file is available when building the application, but it is deleted when you clean the build directory with the ``pristine`` target (see Zephyr's :ref:`zephyr:application_rebuild` for more information).
+This means that this file is available when building the application until you clean the build directory pristinely.
```

Comment:
Would a user know what it means to "clean the build directory pristinely"? I'm not familiar with how often that phrasing is used among developers and just want to confirm.

Maybe it would be clearer to refer to the action like:
```suggestion
This means that this file is available when building the application until you run a pristine build action to clean the build directory.
```

Reply:
This phrase comes from the PR #6121 , more exactly from a series of comments that culminated with the following one: https://github.com/nrfconnect/sdk-nrf/pull/6121#discussion_r764762082 So I would prefer not to touch these lines further, as my edit does not change the meaning that much, while your comment (run a build vs clean the dir) might require further checks.